application-xpoll-ui/src/main/resources/XPoll/XPollClass.xml

```diff
@@ -117,22 +128,34 @@
       <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
     </status>
     <type>
+      <cache>0</cache>
       <customDisplay/>
-      <defaultValue>0</defaultValue>
+      <defaultValue/>
       <disabled>0</disabled>
-      <displayFormType>select</displayFormType>
-      <displayType>1/0</displayType>
+      <displayType>select</displayType>
+      <freeText/>
+      <hint/>
+      <largeStorage>0</largeStorage>
+      <multiSelect>0</multiSelect>
       <name>type</name>
       <number>6</number>
+      <picker>1</picker>
       <prettyName>Type</prettyName>
+      <relationalStorage>0</relationalStorage>
+      <separator> </separator>
+      <separators>|, </separators>
+      <size>1</size>
+      <sort/>
       <unmodifiable>0</unmodifiable>
       <validationMessage/>
       <validationRegExp/>
-      <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      <values>2|1|0</values>
```

Comment:
Should these values be added too?

Reply:
Are they added by default? I wanted to be sure that it was backwards compatible with old version of the XPollClass - where the field was boolean and set to have the values "1/0".